com/metacube/DSA/Assignment5/Dictionary.java

```diff
@@ -0,0 +1,152 @@
+package GET2018.com.metacube.DSA.Assignment5;
+
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Map;
+import java.util.TreeMap;
+
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+public class Dictionary implements IDictionary {
+	
+	private Map<String, String> dictionary = new TreeMap<String, String>();
+
+	public Map<String, String> getDictionary() {
+		return dictionary;
+	}
+
+	public Dictionary() {
+		// TODO Auto-generated constructor stub
```

Comment:
Remove this auto-generated stub comment. You can use default constructor to initialize empty dictionary. private Map<String, String> dictionary reference could be at the top as class level variable and you can instantiate it here. Just a suggestion. Not mandatory to implement. This is just a good practice.